### ISSUE TITLE
[Feature Request]  Missing sources notification Improvement

### ISSUE THREAD

**Issue by saud-97** [Jays2Kings/tachiyomiJ2K#1230]
When failed updates show from global updates due to missing sources. The user has to manually figure out each single missing source alone and searching it in the extensions page to find it manually. I suggest creating a view that opens when missing sources error show up during library updates which will show info about the missing sources and a quick install button.

### Why/User Benefit/User Problem
Install button to allow to quickly install messing sources, especially useful after restoring a backup in a new phone.

### What/Requirements
I suggest creating a separate notification (with new notification_ID) for missing sources error. And make it clickable and manually dismissible (i.e., doesn't disappear when clicking it).
Upon opening the notification a view would show up with a list of missing sources, how many manga coming from each source, and the language. And most importantly add an install button to allow to quickly install sources.

**Comment by saud-97** [comment]
I have seen the clear database in advanced settings view where it shows each manga / source. I suggest to reuse that and do minor changes to it.

**Comment by Jays2Kings** [comment]
how often does someone have manga in their library and not the installed source? When restoring you're told then what sources are missing. On top of that, theres no telling if a source isnt available anymore so it would be a rarely seen/used screen

**Comment by saud-97** [comment]
Yes when restoring there is no view for quickly installing missing sources, if you distribute you library across several languages and sources following DDOS mitigations. Users has to manually screenshot the sources list and try to add them 1 by 1, it's quite tedious and even difficult when they are spanned across multi-languages. 

Also, verifying sources availability should already be a method used inside the extensions tab since it would show the full list of extensions that are available for install from tachiyomi-extensions repo.
